addon/types/model_export/model.py

```diff
@@ -172,14 +177,34 @@ def generate_qc(self):
             qc.write('$mostlyopaque')
             qc.write('\n')
 
+        if self.transform_source == 'MANUAL':
+            origin_x = self.origin_x
+            origin_y = self.origin_y
+            origin_z = self.origin_z
```

Comment:
The original code negated Z, yours doesn't; I don't know what's right here tbh (VDC says do it, but I'd like to do my own testing).

Reply:
Ah, a silly error on my part, i think that's a copy-paste error.
I can commit a - onto this now if you want